src/core/model/types/mixed_type.h

```diff
@@ -100,36 +115,62 @@ class MixedType final : public Type {
         return CreateType(type_id, is_null_eq_null_);
     }
 
+    [[nodiscard]] static size_t GetAlignment(TypeId type_id) noexcept {
+        // Could possibly be implemented as a virtual method, however, it's much more incovinient
+        // since virtual method requires an object of type `Type` to be called which is not always
+        // available. Moreover, alignment of a type is used only when implementing MixedType
+        // or working with it, so GetAlignment as a general method for any type seems quite useless.
+        switch (type_id) {
```

Comment:
Why not make it a map?

Reply:
I'm not sure if map is better, do you see any advantages of using it?

Reply:
The intent is clearer if it's an `unordered_map`, since this function defines a mapping. It'll also be slightly easier to add another type (date?).